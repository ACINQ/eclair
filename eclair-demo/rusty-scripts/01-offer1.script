# Simple test that we can commit an HTLC
A:offer 1
B:recvoffer
A:commit
B:recvcommit
A:recvrevoke
B:commit
A:recvcommit
B:recvrevoke
checksync
echo ***A***
A:dump
echo ***B***
B:dump
