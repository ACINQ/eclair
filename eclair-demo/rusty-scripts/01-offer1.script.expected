***A***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs: 1
  Received htlcs:
  SIGNED
REMOTE COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs: 1
  SIGNED
***B***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs: 1
  SIGNED
REMOTE COMMITS:
 Commit 1:
  Offered htlcs: 1
  Received htlcs:
  SIGNED
